app/models/forecast.rb

```diff
@@ -18,8 +21,7 @@ def hit
     if (self.lat.nil? or self.lng.nil?)
       return "error!" #should actually throw an error.
     end
-    keyFile = open(Rails.root.join("app", ".api_key"), "r")
-    apiKey = keyFile.readline
+    apiKey = Figaro.env.forecast_io_key # could also use ENV['forecast_io_key']
```

Comment:
@derekriemer I added a gem called figaro for key management.  There's now a file which is in the `.gitignore` which dictates values for the keys.  I'll send you the file securely soon, but that's where we can can store the api keys locally, and Heroku has a nice interface for figaro as well.